kimchi/src/circuits/lookup/lookups.rs

```diff
@@ -550,3 +550,50 @@ fn lookup_pattern_constants_correct() {
         assert_eq!((pat, pat.max_joint_size()), (pat, max_joint_size as u32));
     }
 }
+
+#[cfg(feature = "wasm_types")]
+pub mod wasm {
+    use super::*;
+
+    #[wasm_bindgen::prelude::wasm_bindgen]
+    impl LookupPatterns {
+        #[wasm_bindgen::prelude::wasm_bindgen]
+        pub fn new(xor: bool, lookup: bool, range_check: bool, foreign_field_mul: bool) -> Self {
```

Comment:
nit: can we actually call this `create()`? Here and elsewhere? This is consistent with `Wire` and also, having a static `new` method without it being a constructor is a bit weird

Reply:
One of the structure (`LookupInfo`) already has a  `create` function which conflicts here. Hence this sub-optimal solution.

There are 2 workarounds:
1. Have the same name (not `create` then) in all the involved types for creation functions
2. Change the name of the existing `create` function to something else (like `create_from_features`)

Reply:
I just implemented 2. But this clearly needs the approval of `proof-systems` gatekeepers.